spec/MongoTransform.spec.js

```diff
@@ -51,7 +50,7 @@ describe('parseObjectToMongoObjectForCreate', () => {
 
   //TODO: object creation requests shouldn't be seeing __op delete, it makes no sense to
   //have __op delete in a new object. Figure out what this should actually be testing.
-  notWorking('a delete op', (done) => {
+  xit('a delete op', (done) => {
```

Comment:
Yep, there were a handful of tests that were marked with `notWorking` from the start. `xit` is just `notWorking` but also adds some useful output when the tests are run.